gradle/build-scan.gradle

```diff
@@ -19,12 +26,24 @@ buildScan {
         buildScan.server = 'https://gradle-enterprise.elastic.co'
     }
 
+    if (nodeName) {
+        link 'System logs', "https://infra-stats.elastic.co/app/infra#/logs?" +
+                "&logFilter=(expression:'host.name:${nodeName}',kind:kuery)"
+        buildFinished {
+            link 'System metrics', "https://infra-stats.elastic.co/app/infra#/metrics/host/" +
```

Comment:
I wondered if the URL would be any clearer broken up, e.g something along these lines? TBH I'm not sure myself.

```groovy
def interval = [
  from: startTime - TimeUnit.MILLISECONDS.convert(5, TimeUnit.MINUTES),
  interval: '%3E%3D1m',
  to: System.currentTimeMillis() + TimeUnit.MILLISECONDS.convert(5, TimeUnit.MINUTES)
]

def params = [
  autoReload: '!f',
  refreshInterval: 5000,
  time: '(' + interval.collect { it }.join(',') + ')'
]

def paramStr = params.collect { it }.join(",")

link 'System metrics', "https://infra-stats.elastic.co/app/infra#/metrics/host/" +
    "${nodeName}?_g=()&metricTime=(${paramStr})"
```